Source/Charts/Renderers/YAxisRenderer.swift

```diff
@@ -387,6 +387,52 @@ open class YAxisRenderer: AxisRendererBase
             }
         }
         
+        renderLimitLineFill(context: context)
+        
         context.restoreGState()
     }
+    
+    open func renderLimitLineFill(context : CGContext) {
```

Comment:
`open func renderLimitLineFill(in context : CGContext)`

Reply:
done